graylog2-web-interface/src/views/logic/queries/Query.js

```diff
@@ -32,6 +32,22 @@ export type ElasticsearchQueryString = {
 
 export const createElasticsearchQueryString = (query: string = ''): ElasticsearchQueryString => ({ type: 'elasticsearch', query_string: query });
 
+const _streamFilters = (selectedStreams: Array<string>): Array<{ type: string, id: string }> => {
+  return selectedStreams.map(stream => ({ type: 'stream', id: stream }));
+};
+
+export const filtersForQuery = (streams: ?Array<string>) => {
```

Comment:
I am not so sure where the best place for this helper function is. I can't use the `QueriesFilterStore`, due to a dependency cycle.